api/can_api_v2_definition.py

```diff
@@ -19,6 +19,19 @@ class TestPositivityRatioMethod(GetByValueMixin, enum.Enum):
     OTHER = "other"
 
 
+class MetricSource(GetByValueMixin, enum.Enum):
+    """The data source of a metric. This enumeration lists the places from which CAN fetches
+    data. The source is tracked on a per metric and region timeseries basis."""
```

Comment:
In the API we have "metrics" and "actuals" where metrics are our ~5 core metrics (daily new cases, infection rate, test positivity, etc.). So perhaps rename this (and tweak comment) to "FieldSource" ?

Reply:
Done in 98ca0c5e